admin/app/views/spree/admin/shared/_index_table.html.erb

```diff
@@ -27,7 +27,7 @@
 
       <%= render 'spree/admin/shared/index_table_options', collection: paginated_collection %>
     <% else %>
-      <%= render 'spree/admin/shared/no_resource_found', new_object_url: new_object_url %>
+      <%= render 'spree/admin/shared/no_resource_found', new_object_url: (new_object_url if can?(:create, model_class)) %>
```

Comment:
@mad-eel we don't need this check, we already check permissions in no_resource_found partial

Reply:
I know, but it breaks because there's no new resource path